docs/deployment.md

```diff
@@ -1,24 +1,39 @@
 # Deployment
 
-All ServiceControl deployment options listed below rely on the files produced by [packaging](packaging.md).
+All ServiceControl deployment options listed below rely on the files produced by [packaging](packaging.md). During packaging, deployment binaries are created in the `deploy` directory, some of which are packaged into zip files in the `zips` folder, and other deployment artifacts are built from these pieces.
 
 ## ServiceControl installer
 
-A [windows installer is provided for download](https://docs.particular.net/servicecontrol/installation) containing both ServiceControl Management Utility (SCMU) and [Powershell commands](https://docs.particular.net/servicecontrol/powershell).
+The zips are packaged as embedded resources in the ServiceControl Management Utility (the "installer").

```

Comment:
As mentioned in the packaging doc, they are actually embedded in the installer assembly, not directly in the SCMU exe. Does this matter for this documentation? Not sure,

Reply:
Yeah just meant to mean "In there somewhere" and I can't think of a better way to say it without repeating everything in packaging.md.

Reply:
Maybe this could just say that SCMU is self-contained single exe that has everything inside it?

Reply:
This good?
```suggestion
The zips are packaged as embedded resources in the ServiceControl Management Utility. Although technicall embedded in a specific assembly, ServiceControl Management is shipped as a self-contained, single-file executable that has everything inside it.
```